_1327/templates/base.html

```diff
@@ -48,6 +48,13 @@
 						{% endif %}
 					</li>
 					{% endfor %}
+					{% if CAN_CREATE_INFORMATIONPAGE %}
+						<li class="pull-right">
+							<a href="{% url 'information_pages:create' %}">
+								<span class="glyphicon glyphicon-plus" aria-hidden="true"></span>
+							</a>
+						</li>
+					{% endif %}
```

Comment:
i would say it's correct to have this here for info pages only. the only other types planned for now are transcripts and polls. both of them should be somehow collected in a list on specific pages (a list of transcripts and a list of polls) and the option to add a new page of this type can then be added on these pages.